AsyncDisplayKit/ASCollectionViewLayoutFacilitatorProtocol.h

```diff
@@ -25,6 +25,27 @@
  */
 - (void)collectionViewEditingSectionsAtIndexSet:(NSIndexSet *)indexes;
 
+/**
+ * Inform that the collectionView is adding some cell updates into a batch,
+ * and will perform these batch updates at a later point
+ *
+ * NOTE: used in combination with -collectionViewWillPerformBatchUpdates
+ */
+- (void)collectionViewBatchingCellEditsAtIndexPaths:(NSArray *)indexPaths;
```

Comment:
Disregard this. Actually combine `collectionViewEditingSectionsAtIndexSet`and `collectionViewBatchingSectionEditsAtIndexes` with a batched bool.